topk-js/index.d.ts

```diff
@@ -0,0 +1,195 @@
+/* auto-generated by NAPI-RS */
+/* eslint-disable */
+export declare class Client {
+  constructor(config: ClientConfig)
+  collections(): CollectionsClient
+  collection(name: string): CollectionClient
+}
+
+export declare class CollectionClient {
+  get(id: string, fields?: Array<string> | undefined | null, lsn?: number | undefined | null, consistency?: ConsistencyLevel | undefined | null): Promise<Record<string, any>>
+  count(lsn?: number | undefined | null, consistency?: ConsistencyLevel | undefined | null): Promise<number>
+  query(query: query.Query, lsn?: number | undefined | null, consistency?: ConsistencyLevel | undefined | null): Promise<Array<Record<string, any>>>
+  upsert(docs: Array<Record<string, any>>): Promise<number>
+  delete(ids: Array<string>): Promise<number>
+}
+
+export declare class CollectionsClient {
+  list(): Promise<Array<Collection>>
+  create(name: string, schema: Record<string, schema.FieldSpec>): Promise<Collection>
+  get(name: string): Promise<Collection>
+  delete(name: string): Promise<void>
+}
+
+export interface ClientConfig {
+  apiKey: string
+  region: string
+  host?: string
+  https?: boolean
+  headers?: Record<string, string>
```

Comment:
how are you using headers?

Reply:
removing it, but eventually we may want to allow users to add their custom headers